src/installer/steps/configure/chroot_conf.rs

```diff
@@ -182,6 +183,10 @@ impl<'a> ChrootConfigurator<'a> {
             self.chroot.command("passwd", &[user]).stdin_input(&pass).run()?;
         }
 
+        let dest = self.chroot.path.join(&["home/", user, "/.face"].concat());
+
+        let _ = fs::copy(&profile_icon, &dest);
```

Comment:
Should this error be ignored?

Reply:
Only option to recover would be to remove the file, so I'll do that.